### DISCUSSION TITLE
Mods "Wind Up" and "Wind Down" are not working in-game

### DISCUSSION THREAD

**Discussion by DarioTD** [ppy/osu#14984] [Q&A] [answered]
It used to work in version 2021.927.0 but on version 2021.1006.0 stopped working.

Well kinda
It tries to work but like in milliseconds then it restarts to 1x speed

In short the music pitch does not change in-game it remains a constant 1x speed
Mod "Double Time" works

**Comment by smoogipoo** [comment, marked as answer]
Resolved in https://github.com/ppy/osu/pull/14986
